src/components/Nodes/NodeResources/NodeResources.js

```diff
@@ -1,44 +1,60 @@
-import { UI5Panel } from 'shared/components/UI5Panel/UI5Panel';
 import { UI5RadialChart } from 'shared/components/UI5RadialChart/UI5RadialChart';
 import './NodeResources.scss';
+import { Card, CardHeader } from '@ui5/webcomponents-react';
 
-export function NodeResources({ metrics, headerContent }) {
+export function NodeResources({ metrics }) {
   const { t } = useTranslation();
   const { cpu, memory } = metrics || {};
 
   return (
-    <UI5Panel disableMargin title={headerContent} className="node-resources">
-      {cpu && memory ? (
-        <div className="nodes-workloads__body">
-          <UI5RadialChart
-            color="var(--sapIndicationColor_7)"
-            value={cpu.usage}
-            max={cpu.capacity}
-            title={t('machine-info.cpu-usage')}
-            tooltip={{
-              content: `${t('machine-info.cpu-m')} ${cpu.usage}/${
-                cpu.capacity
-              }`,
-              position: 'bottom',
-            }}
-          />
-          <UI5RadialChart
-            color="var(--sapIndicationColor_6)"
-            value={memory.usage}
-            max={memory.capacity}
-            title={t('machine-info.memory-usage')}
-            tooltip={{
-              content: `${t('machine-info.memory-gib')} ${memory.usage}/${
-                memory.capacity
-              }`,
-              position: 'bottom',
-            }}
-          />
-        </div>
-      ) : (
-        t('components.error-panel.error')
-      )}
-    </UI5Panel>
+    <>
+      <div className="cluster-overview__graphs-wrapper">
+        {cpu && memory ? (
+          <>
+            <Card
+              header={
+                <CardHeader
+                  titleText={t('cluster-overview.statistics.cpu-usage')}
+                />
+              }
+            >
+              <UI5RadialChart
+                color="var(--sapIndicationColor_7)"
+                value={cpu.usage}
+                max={cpu.capacity}
+                tooltip={{
+                  content: `${t('machine-info.cpu-m')} ${cpu.usage}/${
+                    cpu.capacity
+                  }`,
+                  position: 'bottom',
+                }}
+              />
+            </Card>
+            <Card
+              header={
+                <CardHeader
+                  titleText={t('cluster-overview.statistics.memory-usage')}
+                />
+              }
+            >
+              <UI5RadialChart
+                color="var(--sapIndicationColor_6)"
+                value={memory.usage}
+                max={memory.capacity}
+                tooltip={{
+                  content: `${t('machine-info.memory-gib')} ${memory.usage}/${
+                    memory.capacity
+                  }`,
+                  position: 'bottom',
+                }}
+              />
+            </Card>
+          </>
+        ) : (
+          t('components.error-panel.error')
+        )}
+      </div>
+    </>
```

Comment:
As the previous comment.
```suggestion
      </div>
```